common/containers/Tabs/GenerateWallet/components/Keystore/EnterPassword.tsx

```diff
@@ -108,11 +110,11 @@ export default class EnterPassword extends Component<Props, State> {
       const { password, passwordValidation } = this.state;
 
       if (password.length < MINIMUM_PASSWORD_LENGTH) {
-        feedback = `Password must be ${MINIMUM_PASSWORD_LENGTH}+ characters`;
+        feedback = translateRaw('INPUT_PASSWORD_PLACEHOLDER');
       } else if (passwordValidation && passwordValidation.feedback) {
-        feedback = `This password is not strong enough. ${passwordValidation.feedback.warning}.`;
+        feedback = translateRaw('WEAK_PASSWORD') + ' ' + passwordValidation.feedback.warning;
```

Comment:
We should potentially figure out a way to localize these password feedback warnings.